examples/fourier_transform_playground.py

```diff
@@ -146,6 +161,12 @@ def moving_wave(
 viewer.window.add_dock_widget(wdg, area='bottom')
 wdg()
 
-napari.run()
+# wait for the layers to be added before running the viewer
+wait_on_layers()
```

Comment:
```suggestion
wait_for_layers(viewer, ['wave_0'])
```

Corresponding to my change above